src/sinks/websocket/sink.rs

```diff
@@ -261,6 +271,7 @@ impl WebSocketSink {
 
         let bytes_sent = register!(BytesSent::from(Protocol("websocket".into())));
         let events_sent = register!(EventsSent::from(Output(None)));
+        let encode_as_binary = self.should_encode_as_binary(self.encoder.serializer());
```

Comment:
Since the helper is part of `Self`, you can actually drop the argument and on line 243 just do `match self.encoder.serializer())`

Reply:
updated